sql/core/src/test/resources/sql-tests/inputs/query_regex_column.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE TEMPORARY VIEW testData AS SELECT * FROM VALUES
+(1, "1"), (2, "2"), (3, "3"), (4, "4"), (5, "5"), (6, "6")
+AS testData(key, value);
+
+CREATE OR REPLACE TEMPORARY VIEW testData2 AS SELECT * FROM VALUES
+(1, 1), (1, 2), (2, 1), (2, 2), (3, 1), (3, 2)
+AS testData2(a, b);
```

Comment:
Since the test cases are testing the regex pattern matching in column names, could you add more names and let the regex pattern match more columns?